Source/ConversationV1/Models/MessageRequest.swift

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+//Struct for Messages
+public struct MessageRequest: JSONEncodable, JSONDecodable {
```

Comment:
Does this model actually have to be `public`? Does it need to be accessible from client code?

If I understand it correctly, a `MessageRequest` is never instantiated by a client and passed to the `Conversation` service. Instead, the `Conversation` service accepts various parameters and then uses them to internally construct a `MessageRequest`.

If that's the case, can you change the `public` access modifier to `internal` instead?